File: app/src/main/java/com/wadektech/eventshub/ui/MoreInfo.java
package com.wadektech.eventshub.ui;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import androidx.appcompat.app.AppCompatActivity;
import com.wadektech.eventshub.R;


public class MoreInfo extends AppCompatActivity {
    ImageView backbtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_more_info);

        backbtn = findViewById(R.id.backbtn);
        backbtn.setOnClickListener(v -> finish());

        getWindow().getDecorView().setSystemUiVisibility(View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR);
    }
}

File: app/src/main/java/com/wadektech/eventshub/ui/MoreInfo.kt
package com.wadektech.eventshub.ui

import android.os.Bundle
import android.view.View
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.wadektech.eventshub.R

class MoreInfo : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_more_info)
        val backbtn : ImageView= findViewById(R.id.backbtn)

        backbtn.setOnClickListener {
            finish()
        }
        window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
    }
}
